common/src/main/org/jetbrains/kotlinx/lincheck/util/AnalysisSections.kt

```diff
@@ -304,12 +308,6 @@ internal class AnalysisProfile(val analyzeStdLib: Boolean) {
     @Suppress("UNUSED_PARAMETER") // methodName is here for uniformity and might become useful in the future
     fun shouldBeHidden(className: String, methodName: String): Boolean = 
         !analyzeStdLib && (isConcurrentCollectionsLibrary(className) || isCollectionsLibrary(className))
-
-    companion object {
-        val DEFAULT = AnalysisProfile(
-            analyzeStdLib = ManagedCTestConfiguration.DEFAULT_STDLIB_ANALYSIS_ENABLED
-        )
-    }
```

Comment:
Can we keep it here? You can invert the dependencies and instead re-define `ManagedCTestConfiguration.DEFAULT_STDLIB_ANALYSIS_ENABLED = AnalysisProfile.DEFAULT.analyzeStdLib`